examples/bratko_kopec/bratko_kopec.py

```diff
@@ -63,7 +65,6 @@ def test_epd_with_fractional_scores(engine, epd, VariantBoard, movetime):
     score = 0.0
 
     print("%s:" % epd_string, end=" ")
-    sys.stdout.flush()
```

Comment:
Ah, then no. These have to stay because stdout is line-buffered by default and we print progress as parts of a single line, here. Nicer would be `print(..., flush=True)`, but unfortunately that's not supported in Python 2.